superset-frontend/src/SqlLab/components/QueryTable/index.tsx

```diff
@@ -29,133 +28,158 @@ import Button from 'src/components/Button';
 import { fDuration } from 'src/modules/dates';
 import Icons from 'src/components/Icons';
 import { Tooltip } from 'src/components/Tooltip';
+import { Query } from 'src/SqlLab/types';
 import ResultSet from '../ResultSet';
 import ModalTrigger from '../../../components/ModalTrigger';
```

Comment:
this is a carry over from the old file, but while you're here, do you mind changing this import to be 'src/components/ModalTrigger' (absolute instead of relative) since it's going up more than one level? Thanks!

Reply:
Thank you Elizabeth for your suggestions. I will work on them later this week.

Reply:
updated on line 32. Thank you